uast/uast.go

```diff
@@ -220,6 +220,16 @@ const (
 	AssignmentVariable
 	AssignmentValue
 
+	// Augmented assignment: +=, -+, *=, etc
```

Comment:
Try to follow the style of other comments. For example, in th is case it wold be something like this:

```
// AugmentedAssignment is an augmented assignment (e.g. +=, -=, *=).
// It is expected that children contains, at least, an AugmentedAssignmentOperator, an AugmentedAssignmentVariable
// and AugmentedAssignmentValue.
AugmentedAssignment
```